src/xERC20/contracts/XUltraLRT.sol

```diff
@@ -542,7 +549,7 @@ contract XUltraLRT is
 
         // minted lrt
         uint256 mintedLRT = ultraLRT.balanceOf(address(this)) - ultraLRTAmount;
-
+        emit LRTMinted(_amount, mintedLRT);
```

Comment:
maybe name this slighly more descriptively? like `L1LRTMintedByLockbox`

Reply:
renamed to L1LRTMinted